src/System.Buffers/tests/System.Buffers.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{62E2AD5F-C8D0-45FB-B6A5-AED2C77F198C}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AssemblyName>System.Buffers.Tests</AssemblyName>
+    <LangVersion>6</LangVersion>
+    <StringResourcesPath>..\src\Resources\Strings.resx</StringResourcesPath>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="ArrayPool\UnitTests.cs" />
+    <Compile Include="..\src\System\Buffers\*.cs" />
```

Comment:
The test .csproj should be using a P2P reference to the src .csproj, not including all of the source files.